tests/test_series.py

```diff
@@ -1147,6 +1165,9 @@ def func(s: pd.Series[int]) -> float:
         np.floating,
     )
 
+    # test below passes with mypy but pyright correctly sees it as pd.Series[float]
+    # check(assert_type(s.groupby([1,1,2,2]).agg(lambda x: x.astype(float).min()), pd.Series), pd.Series, float)
```

Comment:
Keep the commented test in there so it is still there and executes, since it works for both type checkers, but comment out the one that is "better" that has `pyright` infer it as `Series[float]`.